pro/src/new_components/VenueForm/Informations/SiretOrCommentFields/__specs__/SiretOrCommentFields.spec.tsx

```diff
@@ -46,9 +53,11 @@ const renderSiretOrComment = async ({
 
   return {
     ...rtlReturns,
-    buttonSubmit: screen.getByRole('button', {
-      name: 'Submit',
-    }),
+    buttonSubmit: await waitFor(() =>
+      screen.getByRole('button', {
+        name: 'Submit',
+      })
```

Comment:
ptet ça marchait comme ça aussi

```suggestion
    buttonSubmit: await screen.findByRole('button', {
        name: 'Submit',
      })
```

Reply:
J'ai encore les warnings si je mets pas le waitFor :(